### ISSUE TITLE
ResetAllButton should use type 'button' in parallel DOM

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#248]
See https://github.com/phetsims/john-travoltage/issues/108.  ResetAllButton is currently represented as
```html
<input type="reset" value="‪Reset All‬" tabindex="0">
```
("Reset All" taken from translatable strings, attributes modeled after http://webaim.org/techniques/forms/controls)

But input of type 'reset' will reset all other peer input element values under the containing 'form' element (if it exists).  We don't want this, values of input elements in the parallel DOM should be linked to axon properties alone.

Screen readers should read the element in the same way if we use a button with an aria-label.  tabindex is also redundant.
```html
<input type="button" aria-label="‪Reset All‬">
```

**Comment by jessegreenberg** [comment]
Done in commit above, closing.
